### PR TITLE
README.md / Static build / Crash fix

### PR DESCRIPTION
**Pull request by uint32**
* Updated README.md with instructions for downloading, installing, and building on Linux. (Solves #3)
* Made some changes to CMakeLists.txt for being able to generate a static build.
* Fixed crash when CA_CreateObject is called without collision data. It will now show an error message.
* Added an error message if CA_RemoveBuilding is called after CA_Init.

### CODE REVIEW

**Comment by Crayder** [comment]
* I'll try to contact P'
* Could you add a Windows build to the CMakeLists?

File: readme.md [hunk 1]
```diff
@@ -1,2 +1,128 @@
...
+		* Please note that you must install the *32bit* (i386) version of the libraries, even if you use a 64bit OS.
+
+##Installing
+
+1. If you haven't done it yet, generate the ColAndreas database (*ColAndreas.cadb*). For doing this, run the Wizard executable (*ColAndreasWizard.exe*) and follow the instructions. Save the file in the folder *scriptfiles/colandreas*
+2. Add the server plugin:
+	* **Windows:** Copy *ColAndreas.dll* to the *plugins* folder of your server and add "ColAndreas" to the *plugins* line in *server.cfg*
+	* **Linux:** Copy *ColAndreas.so* or *ColAndreas_static.so* (see the note above) and add "ColAndreas.so" or "ColAndreas_static.so" to the *plugins* line in *server.cfg*
+3. Copy *ColAndreas.inc* to the folder *pawno/include*
+4. Call `CA_Init()` from `OnGameModeInit` in order to initialize the plugin:
```
Comment (Crayder) [inline readme.md]: This is false. The init' function just loads the default SA map into the server memory.

Comment (uint32) [inline readme.md]: Yeah, I realized later... It has been fixed now.

**Comment by uint32** [comment]
I added Windows support to the CMakeLists, it will generate a Visual Studio solution file with everything ready.

**Comment by Crayder** [comment]
Nice. Pottus is logged on to Mumble, but he's in an admin channel... :/

**Comment by Pottus** [comment]
Ok I merged this pull request.

**Pull request merged**